lib/src/screens/post/ViewPost.dart

```diff
@@ -46,547 +59,530 @@ class ViewPostState extends State<ViewPost> {
-                      decoration: const BoxDecoration(
-                          color: primaryColorSubtle,
-                          borderRadius: BorderRadiusDirectional.vertical(
-                              bottom: Radius.circular(10)),
-                          boxShadow: [
-                            BoxShadow(
-                              blurRadius: 10,
-                              color: Color.fromRGBO(0, 0, 0, 0.1),
-                              offset: Offset(0, 4),
-                            )
-                          ]),
-                      child: Align(
-                          alignment: const AlignmentDirectional(0.675, -0.875),
-                          child: Image.asset(
-                              'assets/images/background_post.png'))),
-                  Padding(
-                    padding: const EdgeInsetsDirectional.all(20),
-                    child: SingleChildScrollView(
-                      child: Column(
-                        mainAxisSize: MainAxisSize.max,
-                        children: [
-                          /// Section-Header
-                          Row(
-                            children: [
-                              Container(
-                                width:
-                                    MediaQuery.of(context).size.width * 0.115,
-                                decoration: const BoxDecoration(
-                                    borderRadius:
-                                        BorderRadius.all(Radius.circular(10)),
-                                    color: Colors.white),
-                                child: IconButton(
-                                  icon: const Icon(
-                                    Icons.arrow_back_ios_new,
-                                    color: textColor3,
-                                  ),
-                                  onPressed: () {
-                                    Navigator.of(context).pop();
-                                  },
-                                ),
-                              ),
-                            ],
-                          ),
+      body: BlocProvider<CommentBloc>(
+        create: (_) => _commentBloc,
+        child: BlocListener<CommentBloc, CommentState>(
+          listener: (context, state) {
+            if (state is DeletingComment) {
+              showDialog(
+                  context: context,
+                  builder: (context) {
+                    return const CupertinoAlertDialog(
+                      title: Text("Deleting Your Comment..."),
+                      // CircularProgressIndicator(color: thirterydColor)),
+                    );
+                  });
+            } else if (state is DeletedComment) {
+              showToast("delete comment success");
+              Navigator.of(context).pop();
+              Navigator.of(context).pop();
+              Navigator.of(context).pop();
```

Comment:
Curious why there are too many Navigator.pop?

Reply:
5555555555555

 this is just a stupid solution for pop from **delete pop-up** to **page View All Post**  when delete success 
 
 I don't know what I need to do about this problem, you can switch to this branch and find a good solution for this! 🥲🙏

Reply:
but you do not have time for this problem, I will goo solution more in tomorrow, or less than